app/api/helpers/csv_jobs_util.py

```diff
@@ -64,9 +66,14 @@ def export_sessions_csv(sessions):
             column.append(strip_tags(session.short_abstract) if session.short_abstract else '')
             column.append(session.created_at if session.created_at else '')
             column.append('Yes' if session.is_mail_sent else 'No')
+            column.append(session.level)
+            column.append(session.state)
+            column.append(session.type)
+            column.append(len(session.long_abstract))
             rows.append(column)
-
-    return rows
```

Comment:
I maybe wrong, but only the missing fields needed to be appended, what does the `Response(stream_with_context` do?

Reply:
There wasn't any csv which was being exported. So, I added that line to stream the csv for it.